docs/13-walkthroughs/1-initializing-a-k8s-cluster.md

```diff
@@ -5,7 +5,7 @@ In this walkthrough, we will demonstrate how to initialize Zarf onto a K8s clust
 ## Prerequisites
 
 - The [Zarf](https://github.com/defenseunicorns/zarf) repository cloned: ([`git clone` Instructions](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository))
-- Zarf binary installed on your $PATH: ([Install Instructions](../3-getting-started.md#installing-zarf))
+- Zarf binary installed on your $PATH: ([Install Instructions](../3-getting-started/index.md#installing-zarf))
```

Comment:
```suggestion
- Zarf binary installed on your $PATH: (Installing Zarf](../3-getting-started/index.md#installing-zarf))
```

Previous place had both lowercase, making consistent

Reply:
Fixed in #1663